.github/workflows/docker-buildx.yml

```diff
@@ -0,0 +1,51 @@
+name: Build and publish docker image
+
+on:
+  workflow_run:
+    workflows: ["Release"]
+    types:
+      - completed
+
+jobs:
+  buildx_and_push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.workflow_run.head_branch }}
+
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
+
+      - name: Get the version
+        id: get_version
+        run: |
+          version=$(git describe --tags --abbrev=0)
+          echo $version
+          echo ${version:1}
+          echo ::set-output name=VERSION::$version
+          echo ::set-output name=VERSION-NO-V::${version:1}
+        shell: bash
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Build and publish docker image
+        if: ${{ !contains(steps.get_version.outputs.VERSION, '-') }}
+        uses: docker/build-push-action@v2
+        with:
+          context: docker/release
+          file: ./docker/release/Dockerfile.release
```

Comment:
We will need to have the folder `docker/release` as in full node

Reply:
ooooh I didn't understand this like that. let me check

Reply:
It basically takes the jar file and the docker folder to build a new image. So we will need to adapt the docker file a bit.

Reply:
Done, I did run a `docker build -f Dockerfile.release .` and then `docker run <image_id>` everything seems fine (I cannot to PG, but it's the same with the official docker image)

Reply:
I will give it a try tomorrow